apps/system/locales/accessibility.en-US.properties

```diff
@@ -336,3 +336,20 @@ accessibility-stateUncheckedAbbr = ( )
 accessibility-statePressedAbbr = (x)
 # LOCALIZATION NOTE: "( )" is a braille abbreviation for unpressed
 accessibility-stateUnpressedAbbr = ( )
+
+# Hint dialogues for actionable events
+accessibility-pushbutton-hint = Double tap to activate
+accessibility-checkbutton-hint = Double tap to activate
+accessibility-combobox-hint = Press and hold options to focus
+accessibility-key-hint = Double tap to activate
+accessibility-link-hint = Double tap to activate
+accessibility-menuitem-hint = Double tap to activate
+#accessibility-'check menu item' = Double tap to activate
+#accessibility-'radio menu item' = Double tap to activate
```

Comment:
Yeah, this one needs to be fixed in gecko, we need to replace ' ' with '' in place where we generate the hint. I'll open a bug. Assume here that spaces are taken out so 'radiomenuitem' etc